web/elm/src/Build/Styles.elm

```diff
@@ -62,6 +62,7 @@ header status =
 body : List (Html.Attribute msg)
 body =
     [ style "overflow-y" "auto"
+    , style "overflow-x" "hidden"
```

Comment:
I don't think it's great to hide the output, either.

Is there a reason you made the change from `word-break: break-all` to `word-break: normal` in the first place? Adding:

```
.timestamped-logs {
  white-space: pre-line;
}
```

seems to fix the horizontal scrolling issue

Reply:
It seems like if we use `work-break: break-all` the output text breaks by characters, so some of the words would be separated by characters on two lines. Yeah, I agree that `white-space: pre-line` would fix this. The `break-word` was meant to be a small improvement here.

🤔  what could go wrong if we hide the horizontal scroll bar here?

Reply:
It's not just hiding the scroll bar - it's hiding all the output that the scroll bar would allow you to see. IMO it's better to break on characters if necessary than to make the parts of the output inaccessible.

https://stackoverflow.com/questions/12699800/smarter-word-break-in-css describes the problem (break words if it can, but if the word itself is still too long, then break it anywhere).

It seems like the solutions involve using `word-break: break-word`, which apparently is deprecated (and has no support in IE), so we should probably shouldn't use it